code/src/components/TaskSummary.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+
+
+export const TaskSummary = () => {
+  const items = useSelector(state => state.tasks.items)
+  const leftTodo = items.filter(item => !item.complete)
+  const totalTodo = items.length
+
+  const [showLeftTodo, setShowLeftTodo] = useState(false)
+
+  return (
+    <SummaryWrapper>
+      <Input>
+        I have {leftTodo.length}/{totalTodo} thing{leftTodo.length === 1 ? ! "" : "s"} left to do ⏱️
+      </Input>
+
+      {showLeftTodo && (
+        <ul>
+          {leftTodo.map(items => (
+            <li key={items.id}>{items.name}</li>
+          ))}
+        </ul>
+      )}
```

Comment:
this is always false and you're not calling setShowLeftTodo anywhere so maybe you should remove this? Or is it something you wanted to implement?